backend/factories.py

```diff
@@ -25,13 +25,13 @@ class ParticipantFactory(BaseFactory):
     class Meta:
         model = Participant
 
-    name = factory.Faker("first_name", locale="pl_PL")
-    lastname = factory.Faker("last_name", locale="pl_PL")
+    first_name = factory.Faker("first_name", locale="pl_PL")
+    last_name = factory.Faker("last_name", locale="pl_PL")
     email = factory.LazyAttribute(
-        lambda obj: "{}@cfp.com".format(obj.lastname)
+        lambda obj: "{}@cfp.com".format(obj.last_name)
```

Comment:
We don't own `cfp.com` domain, it's not connected with us anyhow. We should change domain here to `codeforpoznan.pl` od stick to reserved, testing domains (like `example.com`, or `codeforpoznan.test`).

Reply:
.test is the best option

Reply:
changed to "codeforpoznan.test"